src/index.ts

```diff
@@ -66,32 +64,34 @@ export type CreateTransfer = {
   id: bigint, // u128
   debit_account_id: bigint, // u128
   credit_account_id: bigint, // u128
-  custom_1: bigint, // u128
-  custom_2: bigint, // u128
-  custom_3: bigint, // u128
-  flags: bigint, // u64
-  amount: bigint, // u64
+  user_data: bigint, // u128
+  reserved: Buffer, // [32]u8
   timeout: bigint, // u64, in nano-seconds
+  code: number, // u16 accounting system code to describe the type of transfer (e.g. settlement)
+  flags: number, // u32
+  amount: bigint, // u64,
 }
 
 export enum CreateTransferFlags {
-  accept = (1 << 0),
-  reject = (1 << 1),
-  auto_commit = (1 << 2)
+  linked = (1 << 0),
+  two_phase_commit = (1 << 1),
+  condition = (1 << 2) // whether or not a condition will be supplied
 }
 
 export enum CreateTransferError {
```

Comment:
I think I'd prefer to not have to iterate the entire returned array to check that everything is `ok`.
(Although for linked transfers, I'd technically only need to check the first result.)
Ditto for creating accounts and creating and committing transfers.

Reply:
Thanks @wilsonianb , yes definitely, keeping the 0 value defined in the enum won't change the results behavior, so you'll only get error results as before (you won't need to iterate just to check that everthing is `ok`). It's just that this might come in handy if we need other ways of getting at the full range of enum values (including `ok`).

Reply:
Got it. Yeah, I misunderstood your suggestion.
Thanks for the clarification.